sensors/src/loadbalancer_sensor.py

```diff
@@ -0,0 +1,167 @@
+                    output["server_list"].append(
+                        {
+                            "dataTitle": {
+                                "title_text": "Issue with loadbalancer "
+                                + str(i["loadbalancer_id"] or "null")
+                                + " and amphora "
+                                + str(i["id"] or "null"),
+                                "lb_id": str(i["loadbalancer_id"] or "null"),
+                                "amp_id": str(i["id"] or "null"),
+                            },
+                            "dataBody": {
+                                "lb_status": str(ping_result or "null"),
+                                "amp_status": str(status[1] or "null"),
+                                "lb_id": str(i["loadbalancer_id"] or "null"),
+                                "amp_id": str(i["id"] or "null"),
+                            },
+                        }
+                    )
+                elif status[0].lower() == "error":
+                    output["server_list"].append(
+                        {
+                            "dataTitle": {
+                                "title_text": "Issue with loadbalancer "
+                                + str(i["loadbalancer_id"] or "null"),
+                                "lb_id": str(i["loadbalancer_id"] or "null"),
+                            },
+                            "dataBody": {
+                                "lb_status": str(ping_result or "null"),
+                                "amp_status": str(status[1] or "null"),
+                                "lb_id": str(i["loadbalancer_id"] or "null"),
+                                "amp_id": str(i["id"] or "null"),
+                            },
+                        }
+                    )
+                elif ping_result.lower() == "error":
+                    output["server_list"].append(
+                        {
+                            "dataTitle": {
+                                "title_text": "Issue with amphora "
+                                + str(i["id"] or "null"),
+                                "amp_id": str(i["id"] or "null"),
+                            },
+                            "dataBody": {
+                                "lb_status": str(ping_result or "null"),
+                                "amp_status": str(status[1] or "null"),
+                                "lb_id": str(i["loadbalancer_id"] or "null"),
+                                "amp_id": str(i["id"] or "null"),
+                            },
+                        }
+                    )
+            else:
+                logging.info("%s is fine.", i["id"])
+        if len(output["server_list"]) > 0:
+
+            self.sensor_service.dispatch(
+                trigger="stackstorm_openstack.openstack.loadbalancer", payload=output
+            )
+            return output
```

Comment:
...This `return output` can be dropped as the sensors does not expect us to return anything from `run`